### PR TITLE
Added the total value locked based on the  coin that was staked into …

### PR DESCRIPTION
**Pull request by business-bluemove**
Dear DefiLlama team

We want to send a pull request to update the total value locked value of the $MOVE coin staked into the BlueMove platform. Please note that this TVL excludes the TVL of staked/listed NFT, which is still waiting for a standard calculation for all NFT marketplaces 
Please feel free to refer to the below link: 
https://bluemove.net/rewards/staking-move 
https://twitter.com/BlueMove_OA/status/1631192682486964224 
https://docs.bluemove.net/docs/rewards/move-staking-rewards 

Thanks and Best Regards
BlueMove Team

**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/blue-move-mart exports TVL: 
        
 
 
    aptos                     0
    
    total                    0

File: projects/blue-move-mart/index.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const http =  require('../helper/http')
-const bluemoveApi =  'https://aptos-mainnet-api.bluemove.net/api/market-info/tvl';
+// const bluemoveApi =  'https://aptos-mainnet-api.bluemove.net/api/market-info/tvl';
+const bluemoveApi  = 'https://aptos-mainnet-api.bluemove.net/api/market-info/stake-tvl';
```
Comment (g1nt0ki) [inline projects/blue-move-mart/index.js:3]: I am getting request forbidden for this, is there any way to get this on chain? 
also, can you pull the latest code and count this under `staking` instead?

**Comment by llamatester** [comment]
The adapter at projects/blue-move-mart exports TVL: 
        
 
 
    aptos                     0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/blue-move-mart exports TVL: 
        
 
 
    aptos                     0
    aptos-staking             1.02 M
    staking                   1.02 M
    
    total                    0

**Comment by business-bluemove** [comment]
Hi g1nt0ki

We updated the source code from your request. We did the following changes 
+ Pull the latest code from the master 
+ Calculated TVL from the blockchain instead of calling our internal API
+ Filtered it as the Staking 
Feel free to let me know if you have any questions or concerns 

Thanks

**Comment by g1nt0ki** [comment]
@business-bluemove thanks for fixing it

**Comment by business-bluemove** [comment]
Hi g1nt0ki,

Our TVL still has not been visualized on the TVL dashboard. Did I forget to configure something? Appreciate your help

Thanks!

**Pull request merged**